src/v2/Components/ArtworkGrid/ArtworkGridEmptyState.tsx

```diff
@@ -8,16 +8,14 @@ interface ArtworkGridEmptyStateProps {
 }
 
 export const ArtworkGridEmptyState: React.FC<ArtworkGridEmptyStateProps> = props => (
-  <Box width="100%" my={1}>
+  <Box width="100%">
     <Sticky>
       {({ stuck }) => {
         return (
-          <Box pt={stuck ? 1 : 0}>
+          <Box pt={stuck ? 1 : 0} my={1}>
```

Comment:
This doesn't really address the concern. Golden rule: never use external margins. Find the parent of `ArtworkGridEmptyState` and control spacing there.

Reply:
**question:** The parent in this case is [ArtistArtworkFilter](https://github.com/artsy/force/blob/07afcef957ee741a6635a71ddc86f4b870811232/src/v2/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx#L71) Do you think that something like this would be the correct approach?

```javascript

      ZeroState={() => (
        <Box my={1}>
          <ZeroState />
        </Box>
      )}
```

Reply:
You could spread the props! So `<ZeroState my={1} />`

Reply:
Ah, yeah that's a bit awkward. Though it's definitely improved. A bit better: `ZeroStateProps extends BoxProps`, then you can just do `<ZeroState my={1} />`. It does make me wonder if _all_ `ZeroStates` should handle margins the same way (I see drill down into where it actually gets rendered and apply padding there.

Just to communicate the problem with external margins: it makes component spacing difficult to debug because you don't know what kind of spacing anything you're looking at in code might have. It also impedes reuse since different contexts may require different spacing. So it's just a good general rule to have. Extending `BoxProps` and controlling the margin from the parent is a fine compromise.